api/tests/src/api/applications/test_application_routes.py

```diff
@@ -434,3 +434,63 @@ def test_application_form_get_unauthorized(client, enable_factory_create, db_ses
     )
 
     assert response.status_code == 401
+
+
+def test_application_get_success(client, enable_factory_create, db_session, api_auth_token):
+    application = ApplicationFactory.create(with_forms=True)
+    application_forms = (
+        db_session.execute(
+            select(ApplicationForm).where(
+                ApplicationForm.application_id == application.application_id
+            )
+        )
+        .scalars()
+        .all()
+    )
```

Comment:
Could you not just get it from the relationship you added?
```suggestion
    application = ApplicationFactory.create(with_forms=True)
    application_forms = application.application_forms
```